### PR TITLE
SEC-3.1: Changing the gnmiPort from 6030 to 9339 

### PR DESCRIPTION
**Pull request by stthummuri**
Changing the gnmiPort from 6030 to 9339 since 9339 is the official IANA assigned port for specific gRPC services 'gNMI' and 'gNOI'.

### CODE REVIEW

**Comment by LimeHat** [comment]
Is it reasonable to use a const gnmi port definition in this test?
In all other cases (AFAIK) gnmi layer is provided by ONDATRA and uses a dynamic port binding.

**Comment by greg-dennis** [comment]
Per offline conversation, this will be updated to use an if-statement or switch to use port 9339 for Juniper, until Ondatra raw API updates make hard-coding the port unnecessary.

**Comment by stthummuri** [comment]
> Per offline conversation, this will be updated to use an if-statement or switch to use port 9339 for Juniper, until Ondatra raw API updates make hard-coding the port unnecessary.

Done

File: feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go [hunk 1]
```diff
@@ -55,7 +54,15 @@ func keyboardInteraction(password string) ssh.KeyboardInteractiveChallenge {
 	}
 }
 
-func gnmiClient(ctx context.Context, sshIP string) (gpb.GNMIClient, error) {
+func gnmiClient(ctx context.Context, sshIP string, dut *ondatra.DUTDevice) (gpb.GNMIClient, error) {
+	var gnmiPort int
```
Comment (greg-dennis) [inline feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go:58]: Please just add  "// TODO(greg-dennis): Remove hard-coded gNMI port"

**Review by greg-dennis — APPROVED** [review]

**Review by greg-dennis — DISMISSED** [review]

**Review by sachendras — APPROVED** [review]

**Review by greg-dennis — APPROVED** [review]

**Pull request merged**